travis/test_server.py

```diff
@@ -217,6 +224,8 @@ def main(argv=None):
     install_options = os.environ.get("INSTALL_OPTIONS", "").split()
     expected_errors = int(os.environ.get("SERVER_EXPECTED_ERRORS", "0"))
     odoo_version = os.environ.get("VERSION")
+    instance_alive = str2bool(os.environ.get('INSTANCE_ALIVE'))
+    is_runbot = str2bool(os.environ.get('RUNBOT'))
```

Comment:
Maybe `UNBUFFER` (active by default) instead of `RUNBOT` would be less runbot specific and allow other callers to deactivate `unbuffer` if needed?